test/pcaps/ikev2_nat_t/readme

```diff
@@ -0,0 +1,37 @@
+CONTENTS OF THIS DIRECTORY
```

Comment:
Please don't include a directory, and only a pcap file.

Reply:
Removed